src/components/questionStructure/Task/index.tsx

```diff
@@ -17,6 +18,7 @@ export type TaskProps =
   | CodeTaskProps
   | MCQOneTaskProps
   | MCQMultiTaskProps
+  | HandwritingTaskProps
```

Comment:
We renamed `HandwritingTask` to `ProcessedHandwriting` in #22 when we implemented raw handwriting; this is just like this because when we made it originally it was just known as "HANDWRITING" in the backend.

Therefore, is it ok to leave the renaming to MR #22 to prevent a potentially difficult merge conflict?

Reply:
Yes it is, thank you for clarifying!